src/main/java/seedu/contax/logic/commands/EditPriorityCommand.java

```diff
@@ -14,17 +14,21 @@
 
 
 /**
- * Edits the level of an existing priority in the appointment list with level low, medium or high.
+ * Edits the priority level of an existing priority in the appointment list with level low, medium or high.
```

Comment:
This statement is still incorrect, please read it as a sentence.

Reply:
i try rephrase